### DISCUSSION TITLE
Trying to connect to Wordpress Site: "You did not specify a secret key.

### DISCUSSION THREAD

**Discussion by drdingens** [akeeba/panopticon#782] [Q&A (Community Support)]
Hey
I am trying to connect a Wordpress site and I am getting an error message saying: "You did not specify a secret key."

I provided the API key and other WP sites on the same server are working well. Any ideas?
Thanks
Rolf

![tp3hhEc 1](https://github.com/user-attachments/assets/fceb4ca9-55cf-4e79-89d1-65c84df93e28)

**Comment by drdingens** [comment]
After providing a secret key in Akeeba Backup for WordPress I was able to connect to the Wordpress Site. The message could have been a bit more specific :-)
Cheers
Rolf

**Reply by nikosdion** [reply]
This shouldn't be necessary, but I think I know what is going there. I am going to look into it, as soon as I have some time. We were closed last week, this week is very busy so far.
